hsv_color_ranges:
  - {h: [100, 130], s: [100, 255], v: [20, 255]}
gaussian_blur_kernel_size: 5
do_gaussian_blur: true
do_histogram_equalization: false

### KINECT
# rgb_image_topic: /kinect2/qhd/image_color_rect
# depth_image_topic: /kinect2/qhd/image_depth_rect
# camera_info_topic: /kinect2/qhd/camera_info
# camera_frame: kinect2_rgb_optical_frame

### PRIMESENSE
rgb_image_topic: /camera/rgb/image_rect_color
depth_image_topic: /camera/depth_registered/hw_registered/image_rect
camera_info_topic: /camera/rgb/camera_info
camera_frame: camera_rgb_optical_frame